resources/config.test.ci.edn

```diff
@@ -82,4 +82,5 @@
                                                  :key-deserializer-class-config     "org.apache.kafka.common.serialization.ByteArrayDeserializer"
                                                  :value-deserializer-class-config   "org.apache.kafka.common.serialization.ByteArrayDeserializer"}}
             :tracer               {:enabled         [true :bool]
-                                   :custom-provider ""}}}
+                                   :custom-provider ""}
+            :new-relic            {:report-errors true}}}
```

Comment:
Please set this config to false by default for all non-production environments like test, ci, development, etc.

Reply:
Done